docs/user/integration/loki/README.md

```diff
@@ -0,0 +1,251 @@
+
+### Verify Loki installation
+
+Check that the `loki` Pod has been created in the Namespace and is in the `Running` state:
+
+```bash
+kubectl -n ${K8S_NAMESPACE} get pod -l app.kubernetes.io/name=loki
+```
+
+## Log agent installation
+
+### Install the log agent
+
+To ingest the application logs from within your cluster to Loki, you can either choose an installation based on [Promtail](https://grafana.com/docs/loki/latest/clients/promtail/), which is the log collector recommended by Loki and provides a ready-to-use setup. Alternatively, you can use Kyma's LogPipeline feature based on Fluent Bit.
+
+<div tabs name="default-settings" group="configuration">
+  <details>
+  <summary label="promtail-installation">
+  Install Promtail
+  </summary>
+
+To install Promtail pointing it to the previously installed Loki instance, run:
+
+```bash
+helm upgrade --install --create-namespace -n ${K8S_NAMESPACE} promtail grafana/promtail -f https://raw.githubusercontent.com/kyma-project/telemetry-manager/main/docs/user/integration/loki/promtail-values.yaml --set "config.clients[0].url=https://${HELM_LOKI_RELEASE}.${K8S_NAMESPACE}.svc.cluster.local:3100/loki/api/v1/push"
+```
+  </details>
+  <details>
+  <summary label="fluent-bit-installation">
+  Install Fluent Bit with Kyma's LogPipeline
+  </summary>
+
+>**CAUTION:** This setup uses an unsupported output plugin for the LogPipline.
+
+Apply the LogPipeline:
+
+   ```bash
+   cat <<EOF | kubectl apply -f -
+   apiVersion: telemetry.kyma-project.io/v1alpha1
+   kind: LogPipeline
+   metadata:
+     name: custom-loki
+   spec:
+      input:
+         application:
+            namespaces:
+              system: true
+      output:
+         custom: |
+            name   loki
+            host   ${HELM_LOKI_RELEASE}-headless.${K8S_NAMESPACE}.svc.cluster.local
+            port   3100
+            auto_kubernetes_labels off
+            labels job=fluentbit, container=\$kubernetes['container_name'], namespace=\$kubernetes['namespace_name'], pod=\$kubernetes['pod_name'], node=\$kubernetes['host'], app=\$kubernetes['labels']['app'],app=\$kubernetes['labels']['app.kubernetes.io/name']
+   EOF
+   ```
+
+When the status of the applied LogPipeline resource turns into `Running`, the underlying Fluent Bit is reconfigured and log shipment to your Loki instance is active.
+
+>**NOTE:** The used output plugin configuration uses a static label map to assign labels of a Pod to Loki log streams. It's not recommended to activate the `auto_kubernetes_labels` feature for using all labels of a Pod because this lowers the performance. Follow [Loki's labelling best practices](https://grafana.com/docs/loki/latest/best-practices/) for a tailor-made setup that fits your workload configuration.
+
+  </details>
+</div>
+
+### Verify the setup by accessing logs using the Loki API
```

Comment:
something's wrong with the points' numbering in this doc

Reply:
It just looks weird in the raw text, but each list item being a `1.` is [valid markdown](https://www.markdownguide.org/basic-syntax/#lists-1). Icke prefers it this way.